Tests/TestBase.cs

```diff
@@ -11,7 +10,7 @@ public class TestBase
         internal void MakeSafe()
         {
             // Prevent telemetry data from being reported based on test results
-            RollbarLocator.RollbarInstance.Config.Enabled = false;
+            Utilities._Rollbar.EnableDisable(false);
```

Comment:
Can we rename this method to something more revealing?
E.g. SendTelemetryData

Reply:
Agreed.